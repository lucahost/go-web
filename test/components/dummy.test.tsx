import { render } from '@testing-library/react'
import Dummy from '../../src/components/dummy'

describe('ButtonGroup', () => {
    it('renders correctly', () => {
        const { container } = render(<Dummy text="Hello World" />)
        expect(container.firstChild).toMatchSnapshot()
    })
})
