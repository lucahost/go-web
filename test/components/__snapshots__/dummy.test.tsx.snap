// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonGroup renders correctly 1`] = `
<h1
  class="dummy__PageTitle-sc-1omcpq1-0 bKpmBB"
>
  Hello World
</h1>
`;
